Dockerfile

```diff
@@ -1,20 +1,59 @@
-FROM registry.access.redhat.com/ubi8/ubi
-MAINTAINER Hazelcast, Inc. Management Center Team <info@hazelcast.com>
+ARG MC_VERSION=4.2020.12
+ARG MC_INSTALL_NAME="hazelcast-management-center-${MC_VERSION}"
+ARG MC_INSTALL_JAR="hazelcast-management-center-${MC_VERSION}.jar"
 
-ENV MC_VERSION 4.2020.10
-ENV MC_HOME /opt/hazelcast/management-center
-ENV MC_DATA /data
+FROM alpine:3.12.1 AS builder
```

Comment:
I could not find any info about init containers at [RedHat Certification documentation](https://redhat-connect.gitbook.io/partner-guide-for-red-hat-openshift-and-container/program-on-boarding/certification-workflow) but I have found ubi8 based init container [registry.access.redhat.com/ubi8/ubi-init](https://catalog.redhat.com/software/containers/ubi8/ubi-init/5c359b97d70cc534b3a378c8?container-tabs=overview&gti-tabs=unauthenticated)

I think we should ask to RedHat team, can we use non-RHEL init container or should we use above one?

Reply:
I think that it's just a build image, so we can use whatever we want. Also if it passes the Red Hat scan, then I think it's fine.